modules/RTC/TraceablePeerConnection.js

```diff
@@ -2069,7 +2069,7 @@ TraceablePeerConnection.prototype.close = function() {
  * @private
  */
 const _fixAnswerRFC4145Setup = function(offer, answer) {
-    if (!browser.isChrome()) {
+    if (!browser.isChrome() && !browser.isSafariWithVP8()) {
```

Comment:
tests like this make me wonder if we should name this check something else.  it isn't so much that safari has vp8 here, it's that it's a safari version with more modern support.

Reply:
I agree but I don't know how to name it.